corehq/apps/accounting/tests/test_utils.py

```diff
@@ -0,0 +1,72 @@
+from datetime import date
+
+from django.test import SimpleTestCase
+
+from corehq.apps.accounting.utils import is_date_range_overlapping
+
+
+class TestIsDateRangeOverlapping(SimpleTestCase):
+    def test_first_range_is_contained_in_second_range(self):
+        self.assertTrue(is_date_range_overlapping(date(2025, 1, 3), date(2025, 1, 6),
+                                                  date(2025, 1, 1), date(2025, 1, 10)))
```

Comment:
nit: use `assert` instead of `self.assert...` which has better output with pytest
```suggestion
        assert is_date_range_overlapping(date(2025, 1, 3), date(2025, 1, 6),
                                                  date(2025, 1, 1), date(2025, 1, 10))
```

Reply:
Thanks cb3b71726eb6551f8603312cffe6a769b23a5baa